src/go.mod

```diff
@@ -1,4 +1,4 @@
-module registry-stable
+module github.com/opentofu/registry-stable
```

Comment:
I would suggest creating a redirector subdomain for the module coordinates so we are flexible in terms of Git hosting. ([see this example](https://github.com/ContainerSSH/go.containerssh.io))

Reply:
That's a good idea. We should raise an issue to do something similar in the future. I'll raise one after my lunch

Reply:
I've created https://github.com/spacelift-io/opentofu-registry/issues/2 for this